pkg/pod/list.go

```diff
@@ -12,13 +13,8 @@ import (
 )
 
 // List returns pod inventory in the given namespace.
+// If nsname is empty, returns pods for all namespaces.
```

Comment:
It may be better to add another function that allows for listing in all namespaces such as in the [BMH](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/bmh/list.go#L57) or [CGU](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/cgu/cgulist.go#L13) packages. This way we still get input validation on the list function still

Reply:
Fixed. I missed the existing function.